### ISSUE TITLE
CodeChecker parse error: plist_to_html/static/layout.html No such file or directory

### ISSUE THREAD

**Issue by dazhouzhou** [Ericsson/codechecker#3395]
I install codechecker by pip.
```
pip3 install codechecker
```
I want to get html report.
```
CodeChecker parse --export html --output ./reports_html ./reports
```
error log:
```
Traceback (most recent call last):
  File "/home/xzz/.local/lib/python3.8/site-packages/codechecker_common/cli.py", line 175, in main
    sys.exit(args.func(args))
  File "/home/xzz/.local/lib/python3.8/site-packages/codechecker_analyzer/cmd/parse.py", line 819, in main
    PlistToHtml.HtmlBuilder(context.path_plist_to_html_dist,
  File "/home/xzz/.local/lib/python3.8/site-packages/plist_to_html/PlistToHtml.py", line 194, in __init__
    self._layout = Template(get_file_content(
  File "/home/xzz/.local/lib/python3.8/site-packages/plist_to_html/PlistToHtml.py", line 106, in get_file_content
    with open(file_path, 'r', encoding='utf-8', errors='replace') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/xzz/.local/lib/python3.8/site-packages/plist_to_html/static/layout.html'
```

**Comment by csordasmarton** [comment]
@dazhouzhou this problem will be fixed in the next release by the #3357 PR.

**Comment by csordasmarton** [comment]
I am closing this PR because it is fixed in #3357.

**Comment by pansila** [comment]
I installed 6.18.0, still facing the same issue.
```bash
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/codechecker_common/cli.py", line 206, in main
    sys.exit(args.func(args))
  File "/usr/local/lib/python3.8/dist-packages/codechecker_analyzer/cmd/parse.py", line 410, in main
    html_builder = report_to_html.HtmlBuilder(
  File "/usr/local/lib/python3.8/dist-packages/codechecker_report_converter/report/output/html/html.py", line 134, in __init__
    self._layout = Template(get_file_content(
  File "/usr/local/lib/python3.8/dist-packages/codechecker_report_converter/report/output/html/html.py", line 90, in get_file_content
    with open(file_path, 'r', encoding='utf-8', errors='replace') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/usr/local/lib/python3.8/dist-packages/codechecker_report_converter/report/output/html/static/layout.html'
```
